docs/docs/architecture.mdx

```diff
@@ -13,27 +13,22 @@ responds to a message:
 
 
 
-<img alt="image" src={useBaseUrl("/img/rasa-message-processing.png")} />
+<img caption="Rasa Open Source architecture" src={useBaseUrl("/img/rasa-message-processing.png")}
```

Comment:
This will work, but for raster sources it's better for performance to use the `<Image />` component in the way we discussed in Slack, and also, it's arguably better to place the image next to the doc and reference it relatively. This way it will be versioned along with the docs, and we avoid:

1. a potentially orphaned image in the `static` folder
2. an image which is up-to-date with current docs but no longer the right image for the older versions

Reply:
what do you mean by `place the imge next to the doc`? as in sth like `docs/docs/architecture-img.png`?

Reply:
Yes, that's what I mean 😄 . So then, taking that example, your code, assuming the MDX file is in the same directory, would look something like

```jsx
import archImage from './architecture-img.png';

<Image img={archImage} alt='this text will be invisble, in the alt attribute of the img' caption='this text will be visible, in the figcaption following the img' />
```